pkg/services/services.go

```diff
@@ -67,6 +67,7 @@ func New(logger *slog.Logger, config defs.WalletServices, opts ...func(*options.
 			logger,
 			"MerklePath",
 			servicequeue.NewService1(whatsonchain.ServiceName, arcService.MerklePath),
```

Comment:
It was probably my mistake, but I just spot it, can you please 🥺  change it:
```suggestion
			servicequeue.NewService1(arc.ServiceName, arcService.MerklePath),
```